### ISSUE TITLE
Create a command that can replace the xkcd bot

### ISSUE THREAD

**Issue by smayzes** [mdavis1982/eve#7]
**Commands:** 

`@eve xkcd drop tables`

**Development requirements:**
- Listen for the command xkcd.
- Parse out the strings after "xkcd" and search the xkcd API for a result.
- If we have a result, return the image.
- If no result, return a message to the user to inform them of no results found. 

**Nice to Have**
- Spelling checker, inform the user of the spelling error and offer a suggestion
- If we can return a response to give the user the message `Did you mean “ xkcd”? Yes - No`, upon clicking yes it runs the correct param for them.

**Comment by svenluijten** [comment]
@cwhite92 made xkcdbot, and it's opensource 😄 https://github.com/cwhite92/XKCDBot. You might be able to use that as inspiration.

**Comment by bronxct1** [comment]
This will need to be a little bit more involved since xkcd's api does not have search functionality. We'll either have to perform a web search to find the match or scrape the api and store and search a local data set.

**Comment by svenluijten** [comment]
@bronxct1 which is why I mentioned Chris' https://xkcdbot.com - it searches https://xkcd.com via Bing's API (which is less strict than Google's) and returns & caches the result.

**Comment by cwhite92** [comment]
Yeah feel free to use the endpoint on my bot. From memory I think it's a GET request to https://xkcdbot.com/xkcd?text=compiling

The bot has a local cache of every comic that is checked for updates every 10 minutes. If it can't find the comic by an exact title match, it falls back to a Bing web search and grabs the top xkcd.com result.

If you're worried about relying on another service, it's not going anywhere. It's used in almost 2000 slack teams so I won't take it offline. :)

**Comment by cwhite92** [comment]
Oh and you can also search for the comic ID.

**Comment by cwhite92** [comment]
My bad, looks like it's a POST request not a GET as per Slack's integration requirements. I'm on mobile so couldn't check :)

**Comment by mdavis1982** [comment]
It can be a `POST` request no problem as the request will be coming from Eve itself. If we can use your service @cwhite92 that would be awesome 😄

**Comment by cwhite92** [comment]
Yep that's fine by me.
